astropy/coordinates/builtin_frames/cirs_observed_transforms.py

```diff
@@ -85,22 +88,22 @@ def observed_to_cirs(observed_coo, cirs_frame, coord_type):
 
 @frame_transform_graph.transform(FunctionTransformWithFiniteDifference, CIRS, AltAz)
 def cirs_to_altaz(cirs_coo, altaz_frame):
```

Comment:
So, these functions are no longer needed; you can just do:
```
@frame_transform_graph.transform(FunctionTransformWithFiniteDifference, CIRS, AltAz)
@frame_transform_graph.transform(FunctionTransformWithFiniteDifference, CIRS, HADec)
def cirs_to_observed(cirs_coo, observed_frame):
    ...
```
and the same for `observed_to_cirs`.

Actually, maybe easiest if I just push directly to your branch, then I can do a rebase as well, so the tests can run.
def

Reply:
Sure, go ahead.